### ISSUE TITLE
Icon Request: Snow Flake

### ISSUE THREAD

**Issue by ErraticFox** [feathericons/feather#852]
## Icon Request

* Icon name: Snowflake
* Use case: weather icons
* Screenshots of similar icons: https://images.app.goo.gl/3YnYSfpGfribiSJa9

There's a snow cloud icon, but this could be good for just indicating freezing weather

**Comment by tmcgann** [comment]
This is great for weather apps/UIs. I've seen this used in home automation UIs for smart thermostats that have a "cool" mode. A flame/fire icon for "heat" mode is also common.

**Comment by lscheibel** [comment]
Maybe this one?

![image](https://user-images.githubusercontent.com/44374653/78842060-4b968b00-79ff-11ea-90a2-6fc7b9015f8b.png)

**Comment by tmcgann** [comment]
I like that one. When it's that large, it didn't strike me as a snowflake at first, probably because of the gap in the middle, but that's probably not a big concern since these icons are usually smaller?

**Comment by lscheibel** [comment]
Yes but mostly I had a pretty hard time not ending up with one big mess in the middle. Do you think it's readable enough?

**Comment by tmcgann** [comment]
> Yes but mostly I had a pretty hard time not ending up with one big mess in the middle.

That makes sense for sure.

> Do you think it's readable enough?

What if you connected the "crystals" with a shape instead of lines crossing? For example:

![image](https://user-images.githubusercontent.com/1307188/78911398-7c001880-7a43-11ea-9b8a-3f032372d2b1.png)

![image](https://user-images.githubusercontent.com/1307188/78911539-aa7df380-7a43-11ea-8207-1e529d482c80.png)

You're probably on a pretty tight grid there so it may not even be possible to do a circle or hexagon. given the dimensions. Thoughts?

**Comment by lscheibel** [comment]
![image](https://user-images.githubusercontent.com/44374653/78918320-8694cf00-7a90-11ea-800c-145109b7e338.png)

Here are some more options. A and B do get quite busy when viewed at 100% but I like how they read a little better:

![image](https://user-images.githubusercontent.com/44374653/78918623-fb680900-7a90-11ea-9aac-1fa3eb44b110.png)

**Comment by locness3** [comment]
I prefer A.

**Comment by tmcgann** [comment]
I really like what you came up with there! Hard to pick but I think I prefer them in this order: B, A, C.

They all look great, and I like them all for different reasons: 
- I like A larger than smaller. It's a bit too square or diamond-esque with the 4 prominent points. I guess I like hexagonal snowflakes more than rectangular.
- B looks good small or large. It's the most easily recognized as a snowflake in my opinion, and it seems important that a symbol/icon be immediately recognizable.
- I like C smaller than larger. Very simple and elegant in its own right.

**Comment by lscheibel** [comment]
I would also pick B in that case. Should I create a PR for B?

**Comment by tmcgann** [comment]
Sure!

**Comment by lscheibel** [comment]
Actually I think I found a better solution by the material icon:

![image](https://user-images.githubusercontent.com/44374653/79194644-3372a180-7e2d-11ea-9751-75a070c4fd1b.png)

I think this one is much cleaner and still reads fairly well. What do you think?

**Comment by tmcgann** [comment]
I still prefer B.

This last one is definitely simpler; however, snowflakes are inherently unique and ornate so I like the original ones that have a bit more detail for that reason. This feels almost too simple to me. Do the original three feel too busy to you still?

I applaud you for finding a really simple design! Often hard to do.

**Comment by lscheibel** [comment]
I am just very concerned about the visual weight of the other variants. I love the feather theme because it feels very light and in comparison to other icons of this set, A and B are quite heavy.

I do think, that the other ones look visually more appealing and interesting but since this is not a display icon set, that is going to be displayed at larger sizes, I would still choose the new one. Even just to keep the feather icons more consistent.

I also think, that it has more of that feather look but I guess that's a little too subjective :smile: 

![image](https://user-images.githubusercontent.com/44374653/79254483-136ecc80-7e85-11ea-905b-c78b08ef3d6b.png)

Can a maintainer comment on this?

**Comment by tmcgann** [comment]
Fair points. Definitely want consistency within the broader icon set.

**Comment by locness3** [comment]
I agree too.
